routes/ffxiv.py

```
@@ -0,0 +1,38 @@
+from flask import Blueprint, render_template, request, redirect, Response
```

Ruff

`flask.redirect` imported but unused

Remove unused import

(F401)